src/TestFramework/PhpUnit/Adapter/PestAdapterFactory.php

```diff
@@ -104,7 +110,7 @@ public static function create(
                 $projectDir,
                 new JUnitTestCaseSorter()
             ),
-            new ArgumentsAndOptionsBuilder($executeOnlyCoveringTestCases),
+            new ArgumentsAndOptionsBuilder($executeOnlyCoveringTestCases, $filteredSourceFilesToMutate, $mapSourceClassToTestStrategy),
```

Comment:
might be worth a line returns to avoid super long lines?

```suggestion
            new ArgumentsAndOptionsBuilder(
                $executeOnlyCoveringTestCases,
                $filteredSourceFilesToMutate,
                $mapSourceClassToTestStrategy,
            ),
```

(same for the other adapters)